spec/factories/products.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :product do
+    
```

Comment:
Which model do you want to use, products or meals? choose one

Reply:
We have decided to use products